### DISCUSSION TITLE
VueFire id collides with custom id

### DISCUSSION THREAD

**Discussion by MichaHuhn** [vuejs/vuefire#1501] [Questions and Help] [answered]
I'm using the **Realtime Database**.
VueFire keeps track of the **database key** (eg. `-KguCoSMemQZw3JD6EPh`) by storing it as a non enumerable property **`id`** on each database document/object.

That's the description of the VueFire documentation:
https://vuefire.vuejs.org/guide/realtime-data.html#Document-s-id

![image](https://github.com/vuejs/vuefire/assets/69698193/6b8bb5ee-6001-4320-ac1f-d34264b0e7ec)

### Problem:
I want to add a custom id to each database object. The property name should be `id` and the value could be a UUID for example.
However, the **VueFire database `id` now collides with my custom `id`**.

### Question:
Is there a way to rename the VueFire `id` to eg. `dbKey` or
do I need to name my custom `id` differently, eg. `_id`?

**Comment by posva** [comment, marked as answer]
The `id` field must be named `id` for VueFire. You need to name your custom id differently

**Reply by MichaHuhn** [reply]
Thank you for the clarification!
